### PR TITLE
Update file-sync-troubleshoot-sync-errors.md

### PR DESCRIPTION
**Pull request by guptasonia**
Removed the below errors as they no longer exist: ECS_E_SECURITY_DESCRIPTOR_SIZE_TOO_LARGE
                
ECS_E_DATA_INGESTION_WAIT_TIMEOUT
         
ECS_E_SEEDED_SHARE_NOT_FOUND

Edited ECS_E_SYNC_CONSTRAINT_CONFLICT_ON_FAILED_DEPENDEE

# Pull request guidance

Thank you for your contribution. The content team is monitoring the pull requests. Once your pull request is ready for review, please type "#sign-off" in a new comment or add a comment such as "PR is ready for review and merge."

### CODE REVIEW

**Comment by guptasonia** [comment]
PR is ready for review and merge. @khdownie

File: support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+![image](https://github.com/MicrosoftDocs/SupportArticles-docs/assets/139396378/77affe42-53c7-4a90-bfa6-6b1842c7cad5)
```
Comment (khdownie) [inline support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md:1]:
```suggestion

```

File: support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+![image](https://github.com/MicrosoftDocs/SupportArticles-docs/assets/139396378/b325b587-8cb3-4c7a-b6ab-353c4c750adb)
```
Comment (khdownie) [inline support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md:2]:
```suggestion

```

File: support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
+![image](https://github.com/MicrosoftDocs/SupportArticles-docs/assets/139396378/3660563c-0724-4bf4-8d3e-e0baf5bb7ab1)
```
Comment (khdownie) [inline support/azure/azure-storage/file-sync-troubleshoot-sync-errors.md:3]:
```suggestion

```

**Review by khdownie — CHANGES_REQUESTED** [review]

**Comment by guptasonia** [comment]
#sign-off

**Comment by khdownie** [comment]
#sign-off

**Pull request merged**